assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidgetV2/UserCountGraph.js

```diff
@@ -21,9 +22,153 @@
  */
 import PropTypes from 'prop-types';
 
-// eslint-disable-next-line no-unused-vars
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME as CORE_SITE } from '../../../../../googlesitekit/datastore/site/constants';
+import { STORE_NAME as CORE_USER } from '../../../../../googlesitekit/datastore/user/constants';
+import { STORE_NAME as MODULES_ANALYTICS, DATE_RANGE_OFFSET } from '../../../../analytics/datastore/constants';
+import { isZeroReport } from '../../../../analytics/util/is-zero-report';
+import GoogleChart from '../../../../../components/GoogleChart';
+import { extractAnalyticsChartData } from '../../../util';
+import PreviewBlock from '../../../../../components/PreviewBlock';
+import ReportError from '../../../../../components/ReportError';
+import ReportZero from '../../../../../components/ReportZero';
+const { useSelect } = Data;
+
 export default function UserCountGraph( { dimensionName, dimensionValue } ) {
-	return null;
+	const currentEntityURL = useSelect( ( select ) => select( CORE_SITE ).getCurrentEntityURL() );
+	const dateRange = useSelect( ( select ) => select( CORE_USER ).getDateRange() );
+	const dateRangeDates = useSelect( ( select ) => select( CORE_USER ).getDateRangeDates( {
+		compare: true,
+		offsetDays: DATE_RANGE_OFFSET,
+	} ) );
+
+	const args = {
+		startDate: dateRangeDates.startDate,
+		endDate: dateRangeDates.endDate,
+		dimensions: [ 'ga:date' ],
+		metrics: [
+			{
+				expression: 'ga:users',
+				alias: 'Users Over Time',
+			},
+		],
+		orderby: {
+			fieldName: 'ga:date',
+			sortOrder: 'DESCENDING',
+		},
```

Comment:
Why sort by date in reverse order here? That doesn't make sense to me. The results in API responses are normally sorted correctly: If `ga:date` is the sole dimension, they should be by date ascending.

Reply:
Argh, that was testing something with the charts and it stayed in, my bad! 😓